src/Factory.sol

```diff
@@ -17,40 +17,22 @@ contract Factory is IFactory, Owned {
     /// @inheritdoc IFactory
     bool public canUpgrade = true;
 
-    /// @inheritdoc IFactory
-    address public settings;
-
-    /// @inheritdoc IFactory
-    address public events;
-
     /// @inheritdoc IFactory
     address public implementation;
 
     /// @inheritdoc IFactory
     mapping(address accounts => bool exist) public accounts;
 
     /// @notice mapping of owner to accounts owned by owner
-    mapping(address owner => address[] accounts) private ownerAccounts;
+    mapping(address owner => address[] accounts) internal ownerAccounts;
```

Comment:
**Question:** Why the switch from private to internal?

Reply:
I generally try to avoid using `private`. Makes it harder to test and also reduces composability later if we want to build on top of any of these contracts.